saleor/graphql/page/mutations/pages.py

```diff
@@ -78,10 +86,25 @@ def clean_input(cls, info, instance, data):
             error.code = PageErrorCode.REQUIRED
             raise ValidationError({"slug": error})
 
+        if "publication_date" in cleaned_input and "published_at" in cleaned_input:
```

Comment:
This is only a question: Would be simpler, to provide in the description of `published_at` that, if providing `publication_date` and `published_at` in the same mutation, then we will ignore the `publication_date` value?

Reply:
Yep, it'll be simpler but IMO both approaches are correct, and I don't see the reason for changing it right now.